public/v4/apps/excalidraw.yml

```diff
@@ -0,0 +1,77 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: kiliandeca/excalidraw:$$cap_excalidraw_version
+        restart: always
+        environment:
+            BACKEND_V2_GET_URL: https://$$cap_appname-storage.$$cap_root_domain/api/v2/scenes/
+            BACKEND_V2_POST_URL: https://$$cap_appname-storage.$$cap_root_domain/api/v2/scenes/
+            LIBRARY_URL: https://libraries.excalidraw.com
+            LIBRARY_BACKEND: https://us-central1-excalidraw-room-persistence.cloudfunctions.net/libraries
+            SOCKET_SERVER_URL: https://$$cap_appname-room.$$cap_root_domain/
+            STORAGE_BACKEND: 'https'
+            HTTP_STORAGE_BACKEND_URL: 'https://$$cap_appname-storage.$$cap_root_domain/api/v2'
+        depends_on:
+            - $$cap_appname-room
+            - $$cap_appname-storage-backend
+    $$cap_appname-room:
+        image: excalidraw/excalidraw-room
+        restart: always
+    $$cap_appname-storage-backend:
+        image: kiliandeca/excalidraw-storage-backend:$$cap_storage_backend_version
+        restart: always
+        environment:
+            STORAGE_URI: redis://srv-captain--$$cap_appname-redis:6379
+            STORAGE_TTL: 2592000000
+        depends_on:
+            - $$cap_appname-redis
+        ports:
+            - '8080:8080'
```

Comment:
I assume this is not needed as the containerHttpPort is already set to 8080.

Reply:
Yep leaving it